biome.json

```diff
@@ -0,0 +1,190 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+	"vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": false },
+	"files": {
+		"ignoreUnknown": false,
+		"ignore": [
+			"*.txt",
+			"*.snap",
+			"*.wasm",
+			"*.html",
+			"*.md",
+			"*.gitignore",
+			"*.yml",
+			"*.tsbuildinfo",
+			"**/executor/**",
+			"**/vendor/**",
+			"**/dist/**",
+			"**/node_modules/**"
+		]
+	},
+	"formatter": {
+		"enabled": true,
+		"useEditorconfig": true,
+		"formatWithErrors": false,
+		"indentStyle": "tab",
+		"indentWidth": 2,
+		"lineEnding": "lf",
+		"lineWidth": 120,
+		"attributePosition": "auto",
+		"bracketSpacing": true,
+		"ignore": [
+			"**/tsconfig.tsbuildinfo",
+			"**/.gitignore",
+			"**/.yarn",
+			"**/*.yml",
+			"**/*.md",
+			"**/executor/**",
+			"**/vendor/**",
+			"**/.pnp.cjs",
+			"**/.pnp.loader.mjs",
+			"**/lib/**",
+			"**/dist/**",
+			"**/node_modules/**",
+			"**/preview/",
+			"**/packages/core/scripts/",
+			"packages/core/src/wasm-executor/browser-wasm-executor.js",
+			"packages/core/src/wasm-executor/node-wasm-executor.js",
+			"packages/e2e/src/rpc-methods-test-scripts.js",
+			"packages/e2e/blobs/**",
+			"packages/web-test/playwright-report/**"
+		]
+	},
+	"organizeImports": { "enabled": true },
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": false,
```

Comment:
can we enable recommended and disable failing rules?

Reply:
done.